lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -109,7 +109,7 @@ public function bindParam($column, &$variable, $type = ParameterType::STRING, $l
         assert(is_int($column));
 
         if (! isset(self::$_paramTypeMap[$type])) {
-            throw new MysqliException(sprintf("Unknown type: '%s'", $type));
+            throw new MysqliException(sprintf('Unknown type "%s".', $type));
```

Comment:
Should this be `Unknown type, "%s" given.` to be consistent with other exceptions?

Reply:
I think that format should be used when you oppose the first part to the second part ("$type should be a known type, XXX given). I think it is fine as is.

Reply:
Isn't that the case here? I think the above should be something like:

```
Unknown type, "%s" given.
```

or as you said:

```
$type should be a known type, "%s" given.
```